common/src/main/java/com/microsoft/hydralab/common/entity/agent/DeviceCommand.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+package com.microsoft.hydralab.common.entity.agent;
+
+import lombok.Data;
+
+/**
+ * @author zhoule
+ * @date 01/31/2023
+ */
+@Data
+public class DeviceCommand {
```

Comment:
Rename this to DeviceScriptCommand to map the name defined in Yaml, and move this to agent project, under the same package with CommandActionLoader, so these will only be accessible in the agent project.